src/scripts/__tests__/merge-translations.js

```diff
@@ -1,4 +1,4 @@
-import { mergeObjects } from "../merge-translations"
+import mergeObjects from "../../utils/mergeObjects"
```

Comment:
I assume we'll eventually want to migrate these test files as well?

Reply:
(I'm ok if it's not this PR).

Reply:
Yes! will migrate them in next PRs 💪🏼